companies.xml

```diff
@@ -698,4 +698,46 @@
         <address>Foot Locker, Inc., 330 West 34th Street, New York, NY 10001</address>
         <country>US</country>
     </company>
+    <company>
+        <symbol>VWRD</symbol>
+        <name>Vanguard FTSE All-World UCITS ETF</name>
+        <taxNumber>00B3RBWM25</taxNumber>
+        <address>70 Sir John Rogerson's Quay Dublin 2 Ireland</address>
+        <country>IE</country>
+    </company>
+    <company>
+        <symbol>VXUS</symbol>
+        <name>Vanguard Total International Stock ETF</name>
+        <taxNumber>9219097683</taxNumber>
+        <address>P.O. Box 1110, Valley Forge, 19482-1110</address>
+        <country>US</country>
+    </company>
+    <company>
+        <symbol>VTI</symbol>
+        <name>Vanguard Total Stock Market ETF</name>
+        <taxNumber>9229087690</taxNumber>
```

Comment:
```suggestion
        <taxNumber>23-2590839</taxNumber>
```

From https://www.sec.gov/Archives/edgar/data/36405/000093247106001143/index8a12b062006.txt.

Reply:
But then again on http://pdf.secdatabase.com/1422/0000932471-08-001844.pdf, it is stated to be `23-2671505`, so who knows?

Reply:
This is something I also have no idea how to approach. Thinking more about I believe having a "wrong" number (or none at all) here is not really an issue as long as the ETF doesn't pay out dividends, since tax number is used only in the dividend report. But I know some ETFs do pay dividends...